plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/editors/EditorContentFunction.java

```diff
@@ -127,7 +135,7 @@ public void setText(String text) {
         this.text = text;
     }
 
-    public String getProjectName() {
+    private String getProjectName() {
```

Comment:
Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/28/commits/2581f5ddab9d98e149879c6365b9582961715d55